docs/application/dotnet/guides/nui/view.md

```diff
@@ -1,75 +1,133 @@
+
+**Figure: Rotation and Scaling.**
+
+<table style="width:100%">
+<tr>
+<td style="width:50%" align="center">
+<img src="./media/view_Figure_Orientation.gif" width="100%">
+</td>
+<td style="width:50%" align="center">
+<img src="./media/view_Figure_Scale.gif" width="100%">
+</td>
+</tr>
+</table>
 
-```
-TextLabel label = new TextLabel( "Hello World" );
-label.BackgroundColor = Color.Red;
-```
 
-**Figure: TextLabel object with a red background**
+<a name="2"></a>
+## Directional Navigation
+
+NUI also provides directional navigation between `Views` using arrow keys on the keyboard. You can simply set the following properties to specify the next `FocusableViews` for each direction:
+
+- [View.UpFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_UpFocusableView)
+
+- [View.DownFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_DownFocusableView)
+
+- [View.LeftFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_LeftFocusableView)
+
+- [View.RightFocusableView](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html#Tizen_NUI_BaseComponents_View_RightFocusableView)
+
+When an arrow key is pressed and the current `View` has next `FocusableView` for the direction, the system passes focus to the next `FocusableView`. However, if the current `View` does not have the next `FocusableView` for the direction or the next `ForcusableView` is not on the window, the property returns `NULL` and no change occurs.
 
-![TextLabel object with a red background](./media/background_textlabel.png)
 
 <a name="3"></a>
-## Setting the Background Image
+## View Event
 
-You can set a background image of a view:
+`View` provides a variety of `events` that are commonly used by the components. The following table lists the basic events provided by `View`:
 
-```
-View view = new View();
-view.BackgroundImage = "image.png";
+**Table: View events**
+
+| Event               | Description                                               |
+| ------------------- | --------------------------------------------------------- |
+| `KeyEvent`          | Emitted when the key event is received.                   |
+| `TouchEvent`        | Emitted when the touch input is received.                 |
+| `HoverEvent`        | Emitted when the hover input is received.                 |
+| `WheelEvent`        | Emitted when the wheel event is received.                 |
+| `FocusGained`       | Emitted when the control gets the key input focus.        |
+| `FocusLost`         | Emitted when the control loses key input focus.           |
+| `Relayout`          | Emitted after the size has been set on the View during relayout. |
+| `LayoutDirectionChanged` | Emitted when the layout direction property of the View or its parent View is changed. |
```

Comment:
If the suggestion in https://github.com/Samsung/tizen-docs/pull/962#discussion_r353617662 is not apt, then can we rephrase to:

Emitted when the layout direction property of `View` or its parent `View` is changed.

Please verify.

Usage of articles with a noun is not recommended.

Reply:
I think it is better, Thank you.